CHANGELOG.md

```diff
@@ -1,26 +1,32 @@
 # Summoner
 
-1.0.7
+1.1.0
```

Comment:
1.1.0 at once?

Reply:
@willbasky We decided to make version `1.1.0` without `update` command because it introduces breaking change. And we already did a lot for this version 🙂 And going to do even more.